.github/workflows/secrel4prs.yml

```diff
@@ -0,0 +1,70 @@
+name: "SecRel check for PRs"
+
+on:
+  pull_request:
+    branches: [ main, qa, develop ]
+    # Limit to certain PR event types since this action doesn't need to run for every commit
+    types: [ready_for_review, review_requested]
+
+  # Trigger on every code push to main and develop branches
+  push:
+    branches: [ main, qa, develop ]
+
+  # Trigger on all published GitHub Releases
+  release:
+    types: [ "created" ]
+
+  # Trigger manually
+  workflow_dispatch:
+
+env:
+  TARGET_ENV: 'dev'
+
+jobs:
+  publish-images:
+    # only run for the internal repo
+    if: github.repository == 'department-of-veterans-affairs/abd-vro-internal'
+    outputs:
+      vro-images: ${{ steps.publish-images.outputs.images_list }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Determine target environment"
+        run: |
+          echo "ref_name: ${{github.ref_name}}"
+          echo 'TARGET_ENV=dev' >> $GITHUB_ENV
+          echo "TARGET_ENV: ${{ env.TARGET_ENV }}"
```

Comment:
@sethdarragile6 Let's pair to update this to set TARGET_ENV based on github.ref_name. We may have to merge this PR into the default branch first, so that we can run in manually via the web UI.